src/App.vue

```diff
@@ -116,7 +148,10 @@ export default {
     }
   },
   mounted: function() {
-    this._mapSettings = [];
+    //this.setMultiFlatmap();
```

Comment:
Still need these lines?

Reply:
Optional but I keep it there for testing before the map is properly loaded.

Reply:
Yeah good idea